src/main/python/sunburst_visualization/sunburst.py

```diff
@@ -0,0 +1,98 @@
+import plotly.express as px
+import sys
+import json
+import os
+
+class SunBurstVisualization:
+    def __init__(self, project_path):
+
+        f = open(project_path + os.path.sep + "results.json")
+
+        self.result = json.load(f)
+
+        def createsunburst(self, result):
+            self.result=data
+
+        character_ = []
+        parent_ = []
+        values_ = []
+        act_parent = None
+
+        for files in self.result["files"]:
+
+            for k,v in files.items():
```

Comment:
Please change these variable names, we won't know what they mean in 2 or 3 weeks